### PR TITLE
added fullwidth prop to Select

### PR DESCRIPTION
**Pull request by luhis**
Checking the Bulma docs, the is-fullwidth class option is missing on the Select component:
https://bulma.io/documentation/form/general/

I just followed the methodology used on the Control component, I hope this is what we need.

### CODE REVIEW

File: src/components/form/components/select.js [hunk 1]
```diff
@@ -31,6 +32,7 @@ const Select = ({
       className={classnames('select', className, {
         [`is-${calculatedSize}`]: calculatedSize,
         [`is-${color}`]: color,
+        'is-expanded': fullwidth,
```
Comment (RJSonnenberg) [inline src/components/form/components/select.js:35]: Looking at this example from Bulma's documentation (https://bulma.io/documentation/form/general/#form-addons):
```html
<div class="field has-addons">
  <div class="control is-expanded">
    <div class="select is-fullwidth">
      <select name="country">
        <option value="Argentina">Argentina</option>
        <option value="Bolivia">Bolivia</option>
        <option value="Brazil">Brazil</option>
        <option value="Chile">Chile</option>
        <option value="Colombia">Colombia</option>
        <option value="Ecuador">Ecuador</option>
        <option value="Guyana">Guyana</option>
        <option value="Paraguay">Paraguay</option>
        <option value="Peru">Peru</option>
        <option value="Suriname">Suriname</option>
        <option value="Uruguay">Uruguay</option>
        <option value="Venezuela">Venezuela</option>
      </select>
    </div>
  </div>
  <div class="control">
    <button type="submit" class="button is-primary">Choose</button>
  </div>
</div>
```

It appears that the class of the surrounding `control` should be `is-expanded`, but the class of the `select` should be `is-fullwidth`.

**Review by RJSonnenberg — CHANGES_REQUESTED** [review]

File: src/components/form/components/select.js (hunk 1 above)
Comment (luhis) [reply src/components/form/components/select.js]: Ah yes, my mistake. I just copy pasted from the controller.  Updated.

**Comment by kennethnym** [comment]
There is `fullwidth` prop already on the `Control` component that applies the `is-expanded` modifier. Does that not work properly?

**Comment by luhis** [comment]
> There is `fullwidth` prop already on the `Control` component that applies the `is-expanded` modifier. Does that not work properly?

It works, but we also need to use is-fullwidth on the select container to get the select to be full width

**Comment by couds** [comment]
Hi Sorry for the late response, Event that the Bulma docs do not mention the is-fullwidth on the select I saw it on the [scss code](https://github.com/jgthms/bulma/blob/ee8eb6ad0d8b7bfe3ee75624218b216126a7d0a7/sass/form/select.sass#L70) So I will merge this PR.

Thanks for the contribution

**Review by kennethnym — APPROVED** [review]
Completely forgot about this, but LGTM!

**Pull request merged**